src/masternodes/mn_checks.cpp

```diff
@@ -4498,7 +4476,12 @@ ResVal<uint256> ApplyAnchorRewardTxPlus(CCustomCSView &mnview,
             cbValues.begin()->second,
             anchorReward);
 
-    CTxDestination destination = FromOrDefaultKeyIDToDestination(finMsg.rewardKeyType, finMsg.rewardKeyID, KeyType::MNRewardKeyType);
+    CTxDestination destination;
+    if (height < consensusParams.NextNetworkUpgradeHeight) {
+        destination = FromOrDefaultKeyIDToDestination(finMsg.rewardKeyType, finMsg.rewardKeyID, KeyType::MNRewardKeyType);
+    } else {
+        destination = FromOrDefaultKeyIDToDestination(finMsg.rewardKeyType, finMsg.rewardKeyID, KeyType::MNRewardKeyType);
+    }
```

Comment:
there is no difference between the branches. typo?

Reply:
Good spot. Just added to avoid an unintentional fork and just restored the same behaviour.